docker-compose.yml

```diff
@@ -37,23 +37,24 @@ services:
             context: ./article-store
         volumes:
             - ./article-store:/srv/app
+        environment:
+            - ALLOWED_HOSTS=article-store
+            - APP_ENV=dev
+            - APP_SECRET=f6a87c2e031ecadc0d7eb442729435c8
+            - DATABASE_URL=postgres://postgres:@article-store-postgres:5432/postgres
```

Comment:
👍

Reply:
I see `.cfg` files have vanished, so I assume all configuration is performed here :+1: